### PR TITLE
SCIM cred example for Azure AD and Okta

### PR DESCRIPTION
**Pull request by robotdan**
Add basic example for Okta and Azure AD. We can add on from here if we wish.

### CODE REVIEW

File: site/_layouts/doc.liquid [hunk 1]
```diff
@@ -412,7 +412,9 @@
                       <a href="#" class="sub-menu">SCIM <i class="fal fa-chevron-{% if page.url contains "scim" %}up{% else %}down{% endif %} fa-fw"></i></a>
                       <ul>
                         <li {% if page.url == "/docs/v1/tech/core-concepts/scim.html" %}class="active"{% endif %}><a href="/docs/v1/tech/core-concepts/scim">Overview</a></li>
-                        <li {% if page.url == "/docs/v1/tech/premium-features/scim/scim-sdk.html" %}class="active"{% endif %}><a href="/docs/v1/tech/premium-features/scim/scim-sdk">SCIM-SDK Example</a></li>
+                        <li {% if page.url == "/docs/v1/tech/premium-features/scim/scim-sdk.html" %}class="active"{% endif %}><a href="/docs/v1/tech/premium-features/scim/scim-sdk">SCIM-SDK</a></li>
```
Comment (mooreds) [inline site/_layouts/doc.liquid:415]: Can we do these in alphabetical order? We aren't consistent, but that is how identity providers work.

File: site/docs/v1/tech/premium-features/scim/azure-ad.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+navcategory: premium
+---
+
+:sectnumlevels: 0
+
+== Overview
+
+This is a basic example to get started configuring Azure AD as a SCIM client and establish a connection between FusionAuth and Azure AD for provisioning purposes.
+
+For complete documentation, please refer to the Azure AD provisioning documentation.
```
Comment (mooreds) [inline site/docs/v1/tech/premium-features/scim/azure-ad.adoc:14]: Is there a link we can offer to folks?

File: site/docs/v1/tech/premium-features/scim/okta.adoc [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+navcategory: premium
+---
+
+:sectnumlevels: 0
+
+== Overview
+
+This is a basic example to get started configuring Okta as a SCIM client and establish a connection between FusionAuth and Okta for provisioning purposes.
+
+For complete documentation, please refer to the Okta provisioning documentation.
```
Comment (mooreds) [inline site/docs/v1/tech/premium-features/scim/okta.adoc:14]: Is there a link we can offer folks?

File: site/docs/v1/tech/premium-features/scim/okta.adoc [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+To configure the integration, in your Okta Dashboard, find your Application configured for provisioning and click on the *Provisioning* tab of the Application configuration.
+
+Select *Integration* from the left *Settings* navigation menu, and find the required configuration fields.
+
+image::okta-scim-client-credentials.png[Okta SCIM Provisioning Integration,width=1200,role=shadowed]
```
Comment (mooreds) [inline site/docs/v1/tech/premium-features/scim/okta.adoc:20]: We don't need 'role=shadowed' do we?

File: site/docs/v1/tech/premium-features/scim/azure-ad.adoc [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+To configure the integration, in the Microsoft Azure console, navigate to *Azure AD* and click on *Enterprise Applications* and either create a new application, or select the application you will be using for SCIM provisioning. Once you have selected the application, click on *Provisioning* in the left navigation under the *Manage* section.
+
+Here you will find an *Update Credentials* option in the *Manage Provisioning* section at the bottom of the page. Expand the *Admin Credentials* section find the required configuration fields.
+
+.[breadcrumb]#Home -> Azure AD -> Enterprise Applications -> Provisioning -> Update credentials#
+image::azure-ad-scim-client-credentials.png[Azure AD SCIM Provisioning Integration,width=1200,role=shadowed]
```
Comment (mooreds) [inline site/docs/v1/tech/premium-features/scim/azure-ad.adoc:21]: We don't need role=shadowed, do we? I thought that was deprecated.

**Review by mooreds — APPROVED** [review]
A few suggestions.

Thanks for doing the PR process!

File: site/docs/v1/tech/premium-features/scim/okta.adoc (hunk 3 above)
Comment (robotdan) [reply site/docs/v1/tech/premium-features/scim/okta.adoc:14]: Not really, their doc is a loop of endless links. I ended up using 3rd party doc to figure it out. I could can add a link to the root of their doc and let them search.

File: site/docs/v1/tech/premium-features/scim/azure-ad.adoc (hunk 2 above)
Comment (robotdan) [reply site/docs/v1/tech/premium-features/scim/azure-ad.adoc:14]: Not really, their doc is a loop of endless links. I ended up using 3rd party doc to figure it out. I could can add a link to the root of their doc and let them search.

File: site/_layouts/doc.liquid (hunk 1 above)
Comment (robotdan) [reply site/_layouts/doc.liquid:415]: Sure

File: site/docs/v1/tech/premium-features/scim/okta.adoc (hunk 3 above)
Comment (mooreds) [reply site/docs/v1/tech/premium-features/scim/okta.adoc:14]: 🤯

File: site/docs/v1/tech/premium-features/scim/azure-ad.adoc (hunk 2 above)
Comment (mooreds) [reply site/docs/v1/tech/premium-features/scim/azure-ad.adoc:14]: 🤯 😬

File: site/docs/v1/tech/premium-features/scim/okta.adoc (hunk 4 above)
Comment (robotdan) [reply site/docs/v1/tech/premium-features/scim/okta.adoc:20]: I thought it still did something, but I'll confirm and remove if it is dead. Thanks.

File: site/docs/v1/tech/premium-features/scim/azure-ad.adoc (hunk 5 above)
Comment (robotdan) [reply site/docs/v1/tech/premium-features/scim/azure-ad.adoc:21]: I thought it still did something, but I'll confirm and remove if it is dead. Thanks.

**Pull request merged**